src/hotspot/cpu/x86/macroAssembler_x86.hpp

```diff
@@ -213,10 +213,11 @@ class MacroAssembler: public Assembler {
 #ifdef _LP64
   // Support for argument shuffling
 
-  void move32_64(VMRegPair src, VMRegPair dst);
-  void long_move(VMRegPair src, VMRegPair dst);
-  void float_move(VMRegPair src, VMRegPair dst);
-  void double_move(VMRegPair src, VMRegPair dst);
+  // bias in bytes
+  void move32_64(VMRegPair src, VMRegPair dst, Register tmp = rax, int in_stk_bias = 0, int out_stk_bias = 0);
```

Comment:
I was about to ask if you needed to preserve current code semantics - and I see you did that by adding default parameters - I'd check with some HS engineer to see what the code guidelines are here.